platform-controller/src/main/java/org/hobbit/controller/docker/ContainerStateObserver.java

```diff
@@ -24,6 +24,9 @@
  *
  */
 public interface ContainerStateObserver {
+
+    public static int DOCKER_EXITCODE_SIGKILL = 128 + new sun.misc.Signal("KILL").getNumber();
```

Comment:
In general, it is a very good idea to do it that way. However, as far as I know, the sun classes might disappear in the future and it is not recommended to use them. So maybe defining it as `137` with a longer comment why it is this value might be better.